cli/src/cardCommands.js

```diff
@@ -54,6 +54,7 @@ const cardCommands = {
 
     async sendCard(args) {
         let cardFile = args[0];
+        let update = args[1];
```

Comment:
In all the code , i suggest to name it cardCustomization to be consistent with command line